packages/p2p/src/components/buy-sell/buy-sell-form.jsx

```diff
@@ -109,7 +116,18 @@ const BuySellForm = props => {
                 buy_sell_store.form_props.setSubmitForm(submitForm);
 
                 return (
-                    <React.Fragment>
+                    <div>
```

Comment:
```suggestion
                    <div>
```
Is there any reason why we are replacing the fragment with divs here?

Reply:
Thx for notice! Had a problem with flex position in mobile view, fixed it with flex-direction: column

Reply:
> Is there any reason why we are replacing the fragment with divs here?

Thx for notice! Had a problem with flex position in mobile view, fixed it with flex-direction: column